GpsDataCapturer/wear/src/main/res/values-round/strings.xml

```diff
@@ -0,0 +1,4 @@
+<resources>
+    <string name="start_capture">START</string>
+    <string name="stop_capture">STOP</string>
```

Comment:
These strings won't change for round / rectangular watches
We should be able to delete these

Reply:
Done.